core/common/src/main/java/alluxio/worker/dora/DoraWorker.java

```diff
@@ -33,6 +35,15 @@ public interface DoraWorker extends DataWorker, SessionCleanable {
    */
   FileInfo getFileInfo(String fileId, GetStatusPOptions options) throws IOException;
 
+  /**
+   * List status from Under File System.
+   * @param path
+   * @param options
+   * @return list of status
+   * @throws IOException
+   */
+  UfsStatus[] listStatus(String path, ListOptions options) throws IOException;
```

Comment:
any reason that `FileInfo getFileInfo` but listStatus returns `UfsStatus[] listStatus`, ideally would want dora worker to return to the same kind of information, but this may be more a cleanup work, to cleanup the complicated and memory heavy data structures in Alluxio

Reply:
Yes, good idea.
It's better to refine the `listStatus` to be `FileInfo[] listStatus()`.
I would handle this as a TODO.